src/server/server.js

```diff
@@ -5,13 +5,25 @@ import path from 'path';
 
 const numCpus = os.cpus().length;
 
+if (process.env.SERVER_PORT) {} else {
+  process.env['SERVER_PORT'] = '3000';
+}
+
+if (process.env.GRAPHQL_HOST) {} else {
+  process.env['GRAPHQL_HOST'] = 'localhost';
+}
+
+if (process.env.GRAPHQL_PROTOCOL) {} else {
+  process.env['GRAPHQL_PROTOCOL'] = 'http:';
+}
+
 export const options = {
   authKey: jwtSecret,
   logLevel: 1,
   // change this to scale vertically
   workers: 1 || numCpus,
   brokers: 1,
-  port: process.env.SERVER_PORT || 3000,
+  port: process.env.SERVER_PORT,
```

Comment:
The SERVER_PORT env var needs to be defined anyway, if it isn't the build process stops cause they are undefined for later use in the graphQL fetching.